src/components/Sticky/Sticky.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import { themr, ThemedComponentClass } from 'react-css-themr';
+import { STICKY } from '../ThemeIdentifiers';
+import { classNames } from '@shopify/react-utilities/styles';
+import * as baseTheme from './Sticky.scss';
+
+export type Position = 'top' | 'bottom';
+
+export interface Props {
```

Comment:
@HiirenP also please add `componentClass` in the prop, its possible that we would want to pass any class as a prop from emgage-frontend, so it should include that class here

Reply:
resolved